react/components/RadioGroup/index.js

```diff
@@ -10,48 +10,77 @@ class RadioGroup extends React.Component {
   }
 
   render() {
-    const { options, value, name, disabled, hideBorder } = this.props
+    const {
+      options,
+      value,
+      name,
+      disabled,
+      hideBorder,
+      label,
+      testId,
+      size,
+    } = this.props
+    let labelClasses = 'vtex-input__label db mb3 w-100 c-on-base '
 
+    switch (size) {
+      case 'small':
+        labelClasses += 't-small '
+        break
+      case 'large':
+        labelClasses += 't-body '
+        break
+      default:
+        labelClasses += 't-small '
+        break
+    }
```

Comment:
You can use `classnames` for that. Example: 

```js
import classnames from 'classnames'

const large = size === 'large'

const labelClasses = classnames('vtex-input__label db mb3 w-100 c-on-base', {
  't-body': large,
  't-small': !large
})
```

Reply:
Thanks for the input! I implemented your sugestions!